changelog.md

```diff
@@ -3,6 +3,44 @@ layout: default
+
+The following is a brief summary of those edits:
+
+- Established a style guide to impose consistency in spelling, capitalization of roles and repository names, and use of punctuation. 
+
+- Specified a policy for linking to the Standard or any specific portion of it. Any links to the Standard from other documents should point to the latest released version, and should  link by section name, not number, as the numbers tend to change more than the names.
+
+- Established a document archive policy to add a stable copy of each version of the Standard to the repository, starting with the initial IEEE/ISTO V.1.0.0 document.
+
+- Added “Compromise of the packaged software, such as malware embedded in a trusted package” to the list of what is "Out of Scope" for the Standard.
+
+- Corrected/changed several numbering references in the full verification process and “Step 0” in the procedure for checking root metadata.
+
+- Presented the option to use a counter, instead of a nonce in the ECU Version Report, and specified the purpose of the nonce in the step-by-step instructions for preparing this report. 
+
+- Specified that the Director SHOULD check that the nonce or counter in each ECU Version Report has not been used before to prevent a replay of the ECU Version Report. If the nonce or counter is reused the Director SHOULD drop the request. 
+
+- Changed the way steps were referenced in the ECU process for verifying the latest downloaded metadata.
+
+- Clarified that there is no need to download all metadata from the Image repo if the Director indicates there are no new updates to install. 
+
+- Clarified that metadata is required at manufacturing time, and presented a rationale for why preinstalled metadata is needed.  This step enables an ECU to authenticate that a remote repository is legitimate when it first downloads metadata in the field, which can serve as a defense against rollback attacks.
+
+- Moved a Targets metadata check for unrecognized ECU IDs to a more logical place in the series of checks.
+
+- Resolved an  inconsistency in how checking hashes of images is discussed.
+
+- Clarified the manner in which we identify images by their hash to specify that If the Primary has received multiple hashes for a given image binary via the Targets role,  then it SHALL verify every hash for this image. This step is to be performed  even if the image is identified by a single hash as part of its filename. 
+
+- Improved naming of example hashes to align with NIST policy (https://csrc.nist.gov/projects/hash-functions/nist-policy-on-hash-functions) on hash functions. This change was also made to demonstrate that Uptane is not tied to any particular set of algorithms.
+
+- Clarified that Full verification MUST be performed by Primary ECUS and MAY be performed by Secondary ECUs. 
```

Comment:
```suggestion
- Clarified that full verification MUST be performed by Primary ECUs and MAY be performed by Secondary ECUs. 
```

Reply:
Thanks for the corrections, @patrickvacek.

Reply:
Sure! This one is still open and valid, I think.

Generally this looks good to me but it might make sense for Ira or someone with knows what the expectations are here better to review this as well..

Reply:
Yeah. Missed that one. I have no idea why I capitalized that. I absolutely agree about having Ira check. @iramcdonald, will this suffice?

Reply:
For whatever reason this one last suggested change is still lingering. Not a big deal, but I'd prefer to change that before merging.